examples/text_labels_and_annotations/mathtext_fontset_example.py

```diff
@@ -0,0 +1,82 @@
+"""
+========================
+Mathtext Fontset Example
+========================
+
+A simple example showcasing the new *mathtext_fontset* parameter that can 
+change the family of fonts for each individual text element in a plot.
+
+**OBS: The following code will only work on matplotlib version 3.4 and above.**
+"""
+
+import matplotlib.pyplot as plt
+
+plt.figure(figsize=(20, 10))
+
+"""A simple plot for the background"""
+plt.plot(range(11), color="0.9", label=r'Math Font Example')
+
+
+"""
+Here we hold the text, we will mix normal text and math text
+(math text is the region encased by *$*).
+"""
+msg = r"This is a normal text and "
+msg+= r"$this\ is\ a\ text\ in\ math\ mode,\ look,\ I\ can\ do\ this\ "
+msg+= r"\int_{0}^{\infty } x^2 dx !$"
+
+"""
+Now we set the text to be shown, its (x, y) coordinates, it's size
+and the family of fonts for the math text.
+"""
+plt.text(0.2, 6, msg, size=24, mathtext_fontset='dejavusans')
+
+"""
+The parameter *mathtext_fontset* migth be too long for some, so we
+aliased it to *math_font*, the following code will work just as well. 
+"""
+plt.text(0.2, 4, msg, size=24, math_font='dejavuserif')
+
+"""
+Also, we can call *mathtext_fontset* and *math_font* in most places where 
+there is text, like in the title:
+"""
+
+titleMsg = r"$This\ is\ a\ title\ in\ math\ mode,\ look:\ "
+titleMsg+= r"\int_{0}^{\infty } x^2 dx !$"
+
+plt.title(titleMsg, math_font='cm', size=24)
+
+"""
+The currently supported fonts for text in math mode are available in a file
+called *matplotlibrc.template*, see the :ref:`matplotlib rc file
+<customizing-with-matplotlibrc-files>` for details.
+"""
+
+"""
+The chat is good and all, but let's see the results. Noticed how the normal
+text was NOT changed by the *math_font* parameter?
+"""
```

Comment:
Just change this to "Note how the normal text is not changed by the ``math_font`` parameter."